src/components/navbar/navbar.component.scss

```diff
@@ -0,0 +1,3 @@
+.usersAvatarStyle {
+  background-color: transparent;
```

Comment:
So why is the padding removed?

Reply:
In my local Env. The icon was displayed correctly without the padding. So I removed it. This is how it looked.

![image](https://user-images.githubusercontent.com/61249902/113399014-22fc6380-93bd-11eb-9e39-ed475a3c304d.png)
If it is required then I will add it again.

Reply:
This is how it looks now with the changes.
PS: Here padding is removed. Just the BG color property is changed.

![image](https://user-images.githubusercontent.com/61249902/113402726-1c70ea80-93c3-11eb-9928-6a012a1f6851.png)

Reply:
Icon does not look correctly placed to me without padding. Looks moved to the left.

Reply:
@FlorianRappl,
The styling for the user "avatar icon" seems to be fixed as of now. 
![image](https://user-images.githubusercontent.com/61249902/113645445-87147580-96a4-11eb-96be-1ba14b8246db.png)

Should I close this PR?
Thank you.